tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/README.md

```diff
@@ -0,0 +1,30 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Upgrades the `SystemConfig` contracts for the Holocene hardfork across multiple chains in the sepolia superchain.
+
+This upgrades the `SystemConfig` in the
+[v1.8.0-rc.4](https://github.com/ethereum-optimism/optimism/tree/v1.8.0-rc.4) release.
+
+## Pre-deployments
+
+- `SystemConfig` - `0x33b83E4C305c908B2Fc181dDa36e230213058d7d`
```

Comment:
Any messages you can link to so I can verify this address?

Reply:
See also https://github.com/ethereum-optimism/optimism/issues/13009